rails.md

```diff
@@ -55,46 +60,58 @@ Answer the following questions. First, without external resources. Challenge you
 
 3b. Which of the above routes must always be passed params and why?
 
```

Comment:
Yes update needs params since we have to tell the app what we are updating. As well as show and delete.